README.md

```diff
@@ -1,20 +1,36 @@
 # Hangman
 
+<!-- markdownlint-disable no-inline-html -->
```

Comment:
This line here determines what lint rules are to be ignored using [markdownlint](https://github.com/DavidAnson/markdownlint). You may update this as desired later.